pkg/reconciler/common/prune.go

```diff
@@ -32,6 +27,10 @@ import (
 	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/client-go/kubernetes"
 	"knative.dev/pkg/logging"
+	"os"
+	"regexp"
+	"strconv"
+	"strings"
```

Comment:
nit: those imports are misplaced, they should be on the "stdlib" group

Reply:
on save I guess the IDE puts it alphabetically, correcting it.